src/model/mod.rs

```diff
@@ -114,6 +114,9 @@ pub struct CapabilityProperties {
     pub image: String,
     /// The contract ID of this capability
     pub contract: String,
+    /// Optional provider config
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub config: Option<serde_json::Value>,
```

Comment:
the assumption is that serde_json::Value here will be String variant, so I am using Option<String> elsewhere, but I need to look into how this works.

Reply:
Think about how yaml syntax of  something like 
```
config:
    address: 0.0.0.0
```
will be processed here